Yoast/wordpress-seo
# Connect Related keyphrases API call to the redux state and UI

## Context


* Integrates the SEMrush modal content (country selector, keyphrases table etc) component together with the sending and receiving of API requests to/from SEMrush.

## Summary


This PR can be summarized in the following changelog entry:

* Integrates the SEMrush related keyphrases API call with the RelatedKeyphrasesModalContent component and its contents.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Ensure you have a valid SEMrush token present in your database before testing (see https://github.com/Yoast/wordpress-seo/pull/15607 for more information on how to do this).
* Ensure you've run `composer install`, `yarn` and `grunt build`.
* Navigate to a new or pre-existing post in the admin.
* Add a keyphrase (suggestion: `yoast`) and click the 'Get related keyphrases' button.
* See that the modal pops up and contains the upsell message, country selector with button and debug information.
* Select a country code, hit the 'Change Country' button.
* See that the keyphrases table appears with information regarding the keyphrase.
* Change the country to Hungary and click 'Change Country'. See that a 'no results found' message is displayed.
* Switch an additional 8 - 9 times between the various countries and confirm every time until you hit your daily limit (10) of API calls.
* See that a message regarding the limit being reached is displayed.

## UI changes

## Documentation

## Quality assurance
